stackable-certs/src/lib.rs

```diff
@@ -11,7 +11,7 @@
 //!
 //! - `k8s`: This enables various traits and functions to work with
 //!   certificates and Kubernetes secrets.
-//! - `rustls`: This enables interoperability between this crates types
+//! - `webhook`: This enables interoperability between this crates types
```

Comment:
**praise**: This makes more sense now :+1:

Reply:
Yeah it definitely does.

The old name kinda made sense (because it enabled a lot of rustls related dependencies), but the new one actually describes what it is achieving.